Layouts/E.inc

```diff
@@ -157,9 +151,8 @@ ENDM
 Clear_LED_3 MACRO
 ENDM
 
-
 ;**** **** **** **** ****
 ; Inherit base layout
 ;**** **** **** **** ****
-$set(CUSTOM_PWM_SETUP, CUSTOM_LED)
+$set(CUSTOM_PWM_SETUP,CUSTOM_LED)
```

Comment:
Here I would like to be consistent: either we have a space before the brace or not - no hard feelings about this, as long as it's consistent.

Reply:
I prefer spaces after commas